docker_daemon/metadata.csv

```diff
@@ -1,32 +1,32 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
-docker.cpu.limit,gauge,,percent,,"Limit on CPU available to the container, expressed as percentage of a core",0,docker,docker.cpu.limit,
+docker.cpu.limit,gauge,,percent,,"Limit on CPU available to the container, expressed as percentage of a core",0,docker,docker.cpu.limit,cpu
```

Comment:
The `docker_daemon` check is legacy. It'd be better to push these changes to the new (and generic) `container` check

Reply:
The list of metrics in the [container/metadata.csv](https://github.com/DataDog/integrations-core/blob/master/container/metadata.csv) file look to not 100% correlate with this file. So I'll go ahead and revert the changes to this file and update container/metadata.csv in a followup pr.

Reply:
Works for me 👍